### PR TITLE
Added environment variable for all git configurations

### PR DESCRIPTION
**Pull request by makoscafee**
## Description

https://fairwinds.myjetbrains.com/youtrack/issue/FWI-856

## Checklist

* [ ] I have updated the CHANGELOG of any plugins updated
* [ ] I have updated the version file of any plugins updated
* [ ] I have added tests wherever appropriate
* [ ] I have labeled this PR with a label per plugin updated

### CODE REVIEW

**Review by vitorvezani — APPROVED** [review]
LGTM! 💯

File: plugins/ci/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.12.2
```
Comment (rbren) [inline plugins/ci/CHANGELOG.md:3]: should probably be 0.13

File: plugins/ci/pkg/ci/ci.go [hunk 2]
```diff
@@ -416,39 +416,51 @@ func getGitInfo(repoName, baseBranch string) (gitInfo, error) {
 		return info, fmt.Errorf("%v: %v", "Please be sure to run the insights-ci script inside of a valid git repository, with the branch you are scanning checked out", err)
 	}
```
Comment (rbren) [inline plugins/ci/pkg/ci/ci.go:417]: we can probably remove this check, since now the user could just specify everything w/ environment variables

**Review by rbren — APPROVED** [review]
Thanks Barnie!

**Review by ivanfetch-fw — APPROVED** [review]

**Pull request merged**